examples/ChatQnA/deploy/k8s_getting_started.md

```diff
@@ -0,0 +1,76 @@
+# Getting Started
+
+## Introduction
+Kubernetes is an orchestration platform for managing containerized applications, ideal for deploying microservices based architectures like ChatQnA. It offers robust mechanisms for automating deployment, scaling, and operations of application containers across clusters of hosts. Kubernetes supports different deployment modes for ChatQnA, which cater to various operational preferences:
+
+-   **Using GMC ( GenAI Microservices Connector)**: GMC can be used to compose and adjust GenAI pipelines dynamically on kubernetes for enhanced service connectivity and management.
```

Comment:
It's better markdown hygiene to use only one space between the `-` and the first charter of the list item (not three).